src/index.ts

```diff
@@ -497,6 +502,7 @@ class Formsy extends React.Component<FormsyProps, FormsyState> {
       this.setState({
         canChange: true,
       });
+      onValidationComplete();
```

Comment:
Is this related to context, or is it an unrelated bug-fix, or an accident?

Reply:
https://github.com/formsy/formsy-react/pull/198#discussion_r329430935
Yes, when the last input is removed the form was still invalid (if the removed input was invalid)